integration_test/pg/storage_test.exs

```diff
@@ -33,19 +27,10 @@ defmodule Ecto.Integration.StorageTest do
     assert Postgres.storage_up(correct_params) == {:error, :already_up}
   end
 
-  test "storage up (wrong credentials)" do
-    refute Postgres.storage_up(wrong_user) == :ok
```

Comment:
When we add DBConnection the GenServer log message is much improved as the exception is raised by DBConnection so we get to see the message (and a stacktrace). If it is awkward to handle a nice log we can leave it for now.